src/pages/charge/chef-departement/GestionDiplomes/DiplomePathing/DiplomePathing.jsx

```diff
@@ -0,0 +1,188 @@
+import React, { useState, useEffect } from "react"
+import ConsultationDiplome from "./../ConsultationDiplome/ConsultationDiplome"
+import SoumettreVoeux from "@/pages/charge/enseignant/SoumettreVoeux/SoumettreVoeux"
+import { ChevronRight } from "lucide-react"
+import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "@/components/ui/Select"
+import { useLocation } from "react-router-dom"
+import JoyRide from "react-joyride"
+import "./DiplomePathing.css"
+const DiplomePathing = () => {
+    const [diplomes, setDiplomes] = useState([])
+    const [selectedDiplome, setSelectedDiplome] = useState(null)
+    const [selectedSection, setSelectedSection] = useState(null)
+    const [selectedNiveau, setSelectedNiveau] = useState(null)
+    const [selectedSemestre, setSelectedSemestre] = useState(null)
+    const location = useLocation()
+    useEffect(() => {
+        const getDiplomes = async () => {
+            const response = await fetch("http://localhost:8090/api/isimm/distributionCharge/diplome", {
```

Comment:
mat7otesh `http://localhost:8090` instead use the env variable `import.meta.env.VITE_API_URL`

Reply:
I know i'll change them juste 5ater ma3 b9yet l'equipe 3arfouch ye5dmou beha .